Skyline.DataMiner.CICD.Tools.Validator/OutputWriters/HtmlWriter/ResultWriterHtml.cs

```diff
@@ -0,0 +1,199 @@
+﻿namespace Skyline.DataMiner.CICD.Tools.Validator.OutputWriters
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Text;
+
+    using Microsoft.Extensions.Logging;
+
+    using Skyline.DataMiner.CICD.Tools.Validator.OutputWriters.HtmlWriter;
+    using Skyline.DataMiner.CICD.Validators.Common.Model;
+
+    internal class ResultWriterHtml : IResultWriter
```

Comment:
Maybe add a line somewhere to indicate if 'include-suppressed' was enabled or not. Or remove the suppressed stuff when it's disabled.
Currently it's a bit confusing as I see 0 suppressed and would think that there are no errors suppressed (even though it's just disabled).
![image](https://github.com/SkylineCommunications/Skyline.DataMiner.CICD.Validators/assets/97458010/de3051ea-d2f2-4552-8c7d-f5ab88039894)